dev/io.openliberty.security.jakartasec.3.0.internal_fat.common/test-applications/BaseServlet.war/src/oidc/client/base/servlets/BaseOpenIdConfig.java

```diff
@@ -198,6 +198,29 @@ public String getIdTokenSigningAlgorithmsSupported() {
 
     }
 
+    public int getJwksConnectTimeoutExpression() {
+
+        int value = 500;
+        if (config.containsKey(Constants.JWKSCONNECTTIMEOUTEXPRESSION)) {
+            value = getIntValue(Constants.JWKSCONNECTTIMEOUTEXPRESSION, 500);
+        }
+
+        return value;
+
+    }
+
+    public int getJwksReadTimeoutExpression() {
+
+//        int value = 500;
+        int value = Constants.OVERRIDE_DEFAULT_JWKS_CONN_TIMEOUT;
+        if (config.containsKey(Constants.JWKSREADTIMEOUTEXPRESSION)) {
+            value = getIntValue(Constants.JWKSREADTIMEOUTEXPRESSION, 500);
```

Comment:
```suggestion
            value = getIntValue(Constants.JWKSREADTIMEOUTEXPRESSION, Constants.OVERRIDE_DEFAULT_JWKS_CONN_TIMEOUT);
```

Reply:
And the wrong variable name - should be OVERRIDE_DEFAULT_JWKS_READ_TIMEOUT

Reply:
fixing and will add another commit.